README.md

```diff
@@ -88,7 +88,8 @@ Security Features
   * Session keys must be changed on login to prevent session fixation attacks.
   * Session cookies must have HttpOnly and Secure flags set and the SameSite attribute set to 'strict' or 'lax' (which allows external regular links to login).
   * For more information about potential pitfalls see the [OWASP Session Management Cheat Sheet](https://www.owasp.org/index.php/Session_Management_Cheat_Sheet)
-* [ ] Form that change state should use anti CSRF tokens. Anti CSRF tokens can be dropped for internal sites using SameSite session cookies where we are sure all users will be on Firefox 60+. Forms that do not change state (e.g. search forms) should use the 'data-no-csrf' form attribute.
+* [ ] When using cookies for session management, make sure you have CSRF protections in place. The easiest mitigation if you don't need support for older browsers is to use SameSite cookies. If we you can't use SameSite, make sure that forms that change state use anti CSRF tokens, and that forms that do not change state (e.g. search forms) use the 'data-no-csrf' form attribute.
+  * This does not apply to site that use local-storage JWTs for session management, like sites on SSO.
```

Comment:
nit: I think we could word this more clearly assuming the recommendation is:

```
if <using cookies for session management>:
  if <only supporting newer browers with samesite cookies>:
    use samesite cookies (and that's enough)
  else: 
    use anti-csrf tokens (and optionally samesite cookies and referer and origin checks)
else:
  # you don't need CSRF e.g. our JWT from SSO is OK (but makes having strong CSP more important)
```

might be worth linking to MDN and caniuse for SameSite cookie support too

Reply:
+1 for rewriting our checklist as code :)